pos_ticket_send_by_mail/data/email_template_data.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!--Email template -->
+    <record id="email_send_pos_receipt" model="mail.template">
+        <field name="name">Envoyer Reçu</field>
+        <field name="email_from">ticket@${object.user_id.company_id.email.split('@')[1] or ''|safe}</field>
+        <field name="subject">${object.user_id.company_id.name} Reçu ${object.pos_reference.split(' ')[1]}</field>
+        <field name="partner_to">${object.partner_id.id}</field>
+        <field name="model_id" ref="point_of_sale.model_pos_order" />
+        <field name="auto_delete" eval="True" />
+        <field name="report_template" ref="pos_ticket_send_by_mail.action_report_pos_receipt" />
+        <field name="report_name">Ticket ${object.pos_reference}</field>
+        <field name="lang">${object.partner_id.lang}</field>
+        <field name="body_html"><![CDATA[
+            <p>Merci de votre visite !</p>
+            <p>Vous trouverez votre reçu ${object.pos_reference.split(' ')[1]} en pièce-jointe.<p/>
+            <p>Amicalement,</p>
+            <p>L'équipe de la coopérative</p>
+                        ]]>
+        </field>
+    </record>
+
+</odoo>
+
```

Comment:
Can you translate this to english, and create a french translation file with these strings?

Goal: text in code is in english, but we don't lose the translations